www/index.html

```diff
@@ -223,8 +224,9 @@
   </div>
 </div>
 
-<p>
-  Need help?<br>Ask on <a href="https://discord.gg/rust-lang">#beginners in the Rust Discord</a><br>
+<p id="help">
```

Comment:
With the larger changes, I'd like to be able to see a single commit that has the `id="help"` added here and also the CSS changes you applied here.

Reply:
Ok. I will do it.